server/driver_tidb.go

```diff
@@ -306,6 +307,10 @@ func (trs *tidbResultSet) Columns() ([]*ColumnInfo, error) {
 	}
 	var columns []*ColumnInfo
 	for _, v := range fields {
+		if v.ColumnAsName.O == "\\N" {
+			v.ColumnAsName = model.NewCIStr("NULL")
+		}
```

Comment:
I think it's ugly to deal with it as a exception here. Would you consider renaming column here https://github.com/pingcap/tidb/blob/master/plan/logical_plan_builder.go#L433？